### PR TITLE
Fix: #17 Renamed parameter & cl option 'config'

### PR DESCRIPTION
**Pull request by cardinalion**

### CODE REVIEW

File: financeager/cli.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def main():
     sys.exit(run(**_parse_command()))
 
 
-def run(command=None, config=None, verbose=False, **cl_kwargs):
+def run(command=None, config_path=None, verbose=False, **cl_kwargs):
```
Comment (pylipp) [inline financeager/cli.py]: Can you name it `config_filepath`? That's the name of the variable that will be automatically generated by argparse from the `--config-filepath` option.

Comment (pylipp) [reply financeager/cli.py]: Can you update the mention of 'config' in the docstring?

Comment (cardinalion) [reply financeager/cli.py]: > Can you name it `config_filepath`? That's the name of the variable that will be automatically generated by argparse from the `--config-filepath` option.

And delete this line? config_filepath = config_path

Comment (pylipp) [reply financeager/cli.py]: Yes.

**Comment by cardinalion** [comment]
Sorry, wrong branches

File: financeager/period.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 from dateutil import rrule
 from datetime import datetime as dt
 import re
+from abc import ABC, abstractmethod
```
Comment (pylipp) [inline financeager/period.py]: I think you mixed up your branches. Switch to the period-branch, and cherry-pick the period-change commit. Back on the config-path-branch, remove the commit by hard-resetting to the previous one.

**Pull request closed without merging**